releasenotes/notes/barebone-backend-option-675c86df4382a443.yaml

```
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added two parameters to :class:`.GenericBackendV2` to exclude error (`include_errors`) and
+    pulse channel information (`include_channels`) from the construction of the backend. These parameters
```

Still using the old names:
```suggestion
    Added two parameters to :class:`.GenericBackendV2` that, when set to `False`, exclude noise properties (`noise_info`) and
    pulse channel information (`pulse_channels`) from the construction of the backend. These parameters
```

---

Great, thanks - I fixed this. :-)